keyboards/namecard2x4/readme.md

```diff
@@ -0,0 +1,21 @@
+# name_card2x4
+
+日本式名刺サイズのキーボードです（55x91mm）。
+Kailh PCB SocketとWS2812/SK6812に対応しています。
+
+This keyboard size is Japanese namecard size(55x91mm).
+
+![name_card2x4](https://i.imgur.com/Bi9SV9A.jpg)
+
+![name_card2x4 with case](https://i.imgur.com/yTv1vSz.jpg)
+
+
+
+Keyboard Maintainer: [takashiski](https://github.com/takashiski)  
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make name_card2x4:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Could you replace this link block with this, instead: 

```markdown
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Reply:
I updated to new information.